backend/fregepoc/indexers/bitbucket/__init__.py

```diff
@@ -0,0 +1 @@
+from .utils import *  # noqa
```

Comment:
Please, use absolute imports as suggested in PEP8 and import only those objects / modules you intend to use. In the case of `__init__` module I thoroughly recommend defining the `__all__`  variable in this module: https://docs.python.org/3/tutorial/modules.html#importing-from-a-package in order to avoid undesired side-effects.

Reply:
Do you mean something like:
`bitbucket/utils.py`:
```python
__all__ = ["get_next_page", "get_forks_count", "get_clone_url", "get_repo_url", "get_last_commit_hash"]
```
`bitbucket/__init__.py`:
```python
from utils import get_next_page, get_forks_count, get_clone_url, get_repo_url, get_last_commit_hash
```
?
I'd prefer to keep the layout flat, so we can refer to `bitbucket.func` instead of `bitbucket.utils.func`. `utils` is the only submodule of `bitbucket` module.

Reply:
I see. I guess the change you have proposed may do the job. However, I am now wondering whether it is a good design to have a module structure like this? Perhaps, a better architecture choice would have been creating an `utils` package with a `bitbucket` module inside of the `indexers` app? Then, you would have been able to do something like this:
```
from indexers.utils import bitbucket`
```
What do you think about it?

Reply:
Good suggestion 👍 It creates a common namespace for all the indexers, so in the future we could define `utils` for any of them without creating a separate directory each time.

Reply:
One more question from a non-pythonist: as we are defining `__all__` in `utils/bitbucket.py`, we can use `from bitbucket import *` in `utils/__init__.py` without cluttering the namespace, am I right?